classes/PodsInit.php

```diff
@@ -554,12 +556,23 @@ public function register_assets() {
 				'jquery-ui-core',
 				'jquery-ui-sortable',
 				'pods-i18n',
-				'marionette',
+				'pods-marionette',
 				'media-views',
 				'media-models',
 			), PODS_VERSION, true
 		);
 
+		/**
+		 * Page builders.
+		 */
+		if (
+			doing_action( 'elementor/editor/before_enqueue_scripts' ) || // Elementor
+			isset( $_GET['fl_builder'] ) || // Beaver Builder
+			isset( $_GET['et_fb'] ) // Divi
```

Comment:
* :warning: Processing form data without nonce verification.
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 88a5a5e9c1fbc30ee7bab70dd505b957</sub>